<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoTestClass</class>
 <widget class="QWidget" name="videoTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>videoTest</string>
  </property>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playbackBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>playback</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryEventBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>queryEvent</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopPbBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>stopPB</string>
   </property>
  </widget>
  <widget class="QFrame" name="playLbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>491</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginout">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>out</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="StartTimeEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="EndTimeEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
